### PR TITLE
feat: use image load on gifs page

### PR DESCRIPTION
**Pull request by T3kstiil3**
## Description

- [x] - fix: prevent form to be send in URL 🐛
- [x] - feat: highlight current date 📅
- [x] - feat: init user tracking 👮‍♀️
- [x] - feat: detect image load and display skeleton 🕵️‍♂️

## Screenshot

Detect image load and display skeleton 🕵️‍♂️
image

Highlight current date: 
image

Init user tracking 👮‍♀️
image

### CODE REVIEW

File: front/src/firebase/init.js [hunk 1]
```diff
@@ -10,10 +10,12 @@ const firebaseConfig = {
     storageBucket: process.env.VUE_APP_FIREBASE_STORAGE_BUCKET,
     messagingSenderId: process.env.VUE_APP_FIREBASE_MESSAGING_SENDER_ID,
     appId: process.env.VUE_APP_FIREBASE_APP_ID,
+    measurementId: "G-R2TFC8JH1K"
```
Comment (ytvnr) [inline front/src/firebase/init.js:13]: what is that ? can we have a env like above ?

File: front/src/views/Gifs.vue [hunk 2]
```diff
@@ -16,7 +16,13 @@
...
             <div class="gifs__container__list">
                 <div class="gifs__container__list__item" v-for="gif in gifs" :key="gif.id">
-                    <img :src="gif.images.fixed_width.url" :alt="gif.title" />
+                    <img
+                        :src="gif.images.fixed_width.url"
+                        :alt="gif.title"
+                        @load="loadComplete(gif.id)"
+                        :hidden="!loadedGifs.includes(gif.id)"
+                    />
+                    <v-skeleton-loader v-if="!loadedGifs.includes(gif.id)" class="mx-auto" type="image"></v-skeleton-loader>
```
Comment (ytvnr) [inline front/src/views/Gifs.vue:25]: why a v-if here and a :hidden for  ?

File: front/src/views/Schedule.vue [hunk 3]
```diff
@@ -98,31 +101,24 @@ export default {
             defaultGifs: [
                 {
                     day: 'Monday',
```
Comment (ytvnr) [inline front/src/views/Schedule.vue:103]: maybe we can delete this tab and get days from Date ? wdyt ?

File: front/src/views/Schedule.vue [hunk 4]
```diff
@@ -210,5 +207,9 @@ export default {
     .v-card {
         margin-bottom: 12px;
     }
+
+    .highlight {
+        background: #1f7087;
```
Comment (ytvnr) [inline front/src/views/Schedule.vue:212]: there is no way to use a variable here ? :)

File: front/src/firebase/init.js (hunk 1 above)
Comment (T3kstiil3) [reply front/src/firebase/init.js:13]: Oh yes my bad in move it to env file ....

File: front/src/views/Gifs.vue (hunk 2 above)
Comment (T3kstiil3) [reply front/src/views/Gifs.vue:25]: Because a need to load image to catch @load

File: front/src/views/Schedule.vue (hunk 3 above)
Comment (T3kstiil3) [reply front/src/views/Schedule.vue:103]: Yes it can be a refactor ?

File: front/src/views/Schedule.vue (hunk 4 above)
Comment (T3kstiil3) [reply front/src/views/Schedule.vue:212]: Yes ! Its the same color as the datepiker card color, maybe we can use vuetify theme for that 🤔
https://vuetifyjs.com/fr-FR/customization/theme/

Comment (T3kstiil3) [reply front/src/views/Schedule.vue:212]: https://github.com/ytvnr/gif-of-the-day/issues/109

File: front/src/views/Schedule.vue (hunk 3 above)
Comment (ytvnr) [reply front/src/views/Schedule.vue:103]: for sure!

**Pull request merged**